crates/blockchain-tree/src/lib.rs

```diff
@@ -57,3 +57,7 @@ pub mod noop;
 mod state;
 
 use aquamarine as _;
+
+/// Implementation used for benchmarking
+pub use rand;
+pub use criterion;
```

Comment:
you don't need to re-export them, they are already available in the benchmark since it's the same crate

Reply:
yeah thought so, ran into an error doing that, fixed it